metricbeat/tests/system/test_autodiscover_jolokia.py

```diff
@@ -52,3 +52,7 @@ def test_jolokia_jmx(self):
         print(evt)
 
         assert evt["jolokia"]["test"]["gc"]["collection_count"] >= 0
+
```

Comment:
We could add some assertions to check that the autodiscover metadata is actually being added.

Reply:
ahh yes, sorry you pointed out this to me last time and I forgot 😢 Will add it in next commit! Thanks!

Reply:
I only added checks for:
`assert evt['jolokia']['server']['product'] == 'tomcat'` and `assert evt['jolokia']['server']['vendor'] == 'Apache'` here since other fields might change with different setup. Please let me know if I'm missing anything here 😄

Reply:
It looks enough to check for these fields, the purpouse is to check that the autodiscover provider is adding fields.